internal/core/queries/get_in_all.go

```diff
@@ -53,6 +53,7 @@ func (ch GetAllIntegrationQueryHandler) Handle(ctx context.Context, _ mediator.M
 			Vendor:                  v.Vendor,
 			AutoPiDefaultTemplateID: im.AutoPiDefaultTemplateID,
 			RefreshLimitSecs:        v.RefreshLimitSecs,
+			TokenID:                 v.TokenID.Int,
```

Comment:
is it ok for TokenID to be `0` if it is null in the db? Or are we not expecting these to be null eventually?

Reply:
@JamesReate Is that good enough for you?